DigitalLearningSolutions.Web.Tests/Helpers/DisplayStringHelperTests.cs

```diff
@@ -134,25 +134,54 @@ public void ConvertNumberToMonthsString_should_return_months_for_more_than_one()
         }
 
         [Test]
-        public void GetDelegateNameString_returns_expected_name_with_no_first_name()
+        public void GetFullNameForDisplay_returns_expected_name_with_no_first_name()
         {
             // When
-            var result = DisplayStringHelper.GetDelegateNameString(null, "LastName");
+            var result = DisplayStringHelper.GetFullNameForDisplay(null, "LastName");
 
             // Then
             result.Should().Be("LastName");
         }
 
         [Test]
-        public void GetDelegateNameString_returns_expected_name_with_first_name()
+        public void GetFullNameForDisplay_returns_expected_name_with_first_name()
         {
             // When
-            var result = DisplayStringHelper.GetDelegateNameString("FirstName", "LastName");
+            var result = DisplayStringHelper.GetFullNameForDisplay("FirstName", "LastName");
 
             // Then
             result.Should().Be("FirstName LastName");
         }
 
+        [TestCase(null)]
+        [TestCase("")]
+        [TestCase(" ")]
+        public void GetFullNameForSorting_returns_last_name_only_when_first_name_is_not_a_name(string? firstName)

```

Comment:
`...when_first_name_is_null_or_whitespace` would be clearer. Is `932fancv90a` a name? 😛 Guess it depends what platform you're on...